torchchat/cli/cli.py

```diff
@@ -148,6 +148,12 @@ def _add_model_config_args(parser, verb: str) -> None:
             help="Whether to compile the prefill. Improves prefill perf, but has higher compile times.",
         )
 
+    model_config_parser.add_argument(
```

Comment:
after migration we shouldn't need anything special for quantized model right

Reply:
Not sure I follow? The things I'm testing out should work for tensor subclass right?

Reply:
oh I meant that with tensor subclass API, quantized checkpoint should be able to be loaded the same way as normal checkpoint.

the code path of loading a quantized model v.s. quantizing model on the fly might still make sense though, maybe just need to change the naming or something